uber/config.py

```diff
@@ -1092,7 +1092,8 @@ def _make_room_trie(rooms):
 c.ACCESS_OPTS.extend(c.GUEST_ACCESS_LEVEL_OPTS)
 c.ACCESS_VARS.extend(c.GUEST_ACCESS_LEVEL_VARS)
 
-c.ROCK_ISLAND_GROUPS = [getattr(c, group_type.upper()) for group_type in c.ROCK_ISLAND_GROUPS]
+if c.ROCK_ISLAND_GROUPS != ['']:
```

Comment:
Nice catch! I have one quick question: if `c.ROCK_ISLAND_GROUPS` is equal to `['']`, it will stay that way. Is that what we want? Or would we prefer it to be set to an empty list?

This is probably fine because of the way our config files are parsed. However, I usually accomplish list filtering by doing something like this:
```
c.ROCK_ISLAND_GROUPS = [getattr(c, s.upper()) for s in c.ROCK_ISLAND_GROUPS if s or s.strip()]
```

That will filter out any values that are empty strings, or whitespace strings, or `None`.

Reply:
Ah, that looks good and still lets the server start when ROCK_ISLAND_GROUPS doesn't have any values. I'll update the code!